client/modules/IDE/selectors/projects.js

```diff
@@ -0,0 +1,34 @@
+import { createSelector } from 'reselect';
+import differenceInMilliseconds from 'date-fns/difference_in_milliseconds';
+import sortBy from 'lodash/sortBy';
+import reverse from 'lodash/reverse';
+import { DIRECTION } from '../actions/sorting';
+
+const getSketches = state => state.sketches;
+const getField = state => state.sorting.field;
+const getDirection = state => state.sorting.direction;
+
+const getSortedSketches = createSelector(
+  getSketches,
+  getField,
+  getDirection,
+  (sketches, field, direction) => {
+    if (field === 'name') {
+      const sortedSketches = sortBy(sketches, 'name');
+      if (direction === DIRECTION.DESC) {
+        return reverse(sortedSketches);
```

Comment:
This potentially sorts the `sketches` list twice. In practice this probably doesn't matter too much for performance. [`lodash.orderBy`](https://lodash.com/docs/#orderBy) allows the sort order to be specified.